.github/workflows/pr_notifier.yml

```diff
@@ -0,0 +1,24 @@
+on:
+  pull_request:
+  #workflow_dispatch:
+  #schedule:
+  #  - cron: '0 */4 * * *'
+
+jobs:
+  pr_notifier:
+    name: PR Notifier
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up Python 3.8
+        uses: actions/setup-python@v2
+        with:
+          python-version: '3.8'
+          architecture: 'x64'
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          pip install -r ./.github/actions/pr_notifier/requirements.txt
```

Comment:
+1 - the pip installer i linked doesnt support require-hashes afaict - this sees to be the most straightforward